src/logic/currencyValues/store/model/currencyValues.ts

```diff
@@ -1,42 +1,43 @@
-import { Record } from 'immutable'
+import { List, Record } from 'immutable'
 
 export enum AVAILABLE_CURRENCIES {
+  ETH = 'ETH',
   USD = 'USD',
   EUR = 'EUR',
+  AUD = 'AUD',
+  BGN = 'BGN',
+  BRL = 'BRL',
   CAD = 'CAD',
-  HKD = 'HKD',
-  ISK = 'ISK',
-  PHP = 'PHP',
+  CHF = 'CHF',
+  CNY = 'CNY',
+  CZK = 'CZK',
   DKK = 'DKK',
+  GBP = 'GBP',
+  HKD = 'HKD',
+  HRK = 'HRK',
   HUF = 'HUF',
-  CZK = 'CZK',
-  AUD = 'AUD',
-  RON = 'RON',
-  SEK = 'SEK',
   IDR = 'IDR',
+  ILS = 'ILS',
   INR = 'INR',
-  BRL = 'BRL',
-  RUB = 'RUB',
-  HRK = 'HRK',
+  ISK = 'ISK',
   JPY = 'JPY',
-  THB = 'THB',
-  CHF = 'CHF',
-  SGD = 'SGD',
-  PLN = 'PLN',
-  BGN = 'BGN',
-  TRY = 'TRY',
-  CNY = 'CNY',
+  KRW = 'KRW',
+  MXN = 'MXN',
+  MYR = 'MYR',
   NOK = 'NOK',
   NZD = 'NZD',
+  PHP = 'PHP',
+  PLN = 'PLN',
+  RON = 'RON',
+  RUB = 'RUB',
+  SEK = 'SEK',
+  SGD = 'SGD',
+  THB = 'THB',
+  TRY = 'TRY',
   ZAR = 'ZAR',
-  MXN = 'MXN',
-  ILS = 'ILS',
-  GBP = 'GBP',
-  KRW = 'KRW',
-  MYR = 'MYR',
 }
 
-type BalanceCurrencyRecord = {
+export type BalanceCurrencyRecord = {
```

Comment:
this is not a record but record props, please use other model definitions for examples on how to correctly type records